pkg/blueprint/filesystem_customizations.go

```diff
@@ -4,14 +4,227 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"path/filepath"
+	"slices"
+	"strings"
 
 	"github.com/osbuild/images/pkg/datasizes"
 	"github.com/osbuild/images/pkg/pathpolicy"
 )
 
+type PartitioningCustomization struct {
```

Comment:
Maybe put this into a new `partition_customizations.go` ? As they are distinict from the tranditional mountpoint based filesystem customizations?

Reply:
Done.